docs/testing/workspaces-migration-guide.md

```diff
@@ -99,31 +105,23 @@ let contract = worker
     .result;
 ```
 
-Or, if you want to create a subaccount with a certain balance:
-
-```rust title="Deployment - workspaces-rs (with explicit balance)"
-use near_units::parse_near;
+:::danger
+'dev_deploy' can't supply the initial balance since testnet controls this amount in the helper contract which is what we're using to create dev accounts on testnet. So, to make it simple, we don't supply it at all (sandbox included). If someone wants to supply an amount in sandbox, they can grab the root account and do:
```

Comment:
I think it is important to underline that it is possible to supply initial balance for a **subaccount** (and not the root one):

```suggestion
'dev_deploy' can't supply the initial balance since testnet controls this amount in the helper contract which is what we're using to create dev accounts on testnet. So, to make it simple, we don't supply it at all (sandbox included). It is however possible to create a **subaccount** with a certain balance in sandbox, they can grab the root account and do:
```